fee_calculator/apps/viewer/presenters/offence_class_presenters.py

```diff
@@ -32,20 +44,32 @@ def __init__(self, offence_class=None, count=None):
     @property
     @abstractmethod
     def label(self):
+        """
+        The label to be used in a form
+        """
         pass
 
     @property
     @abstractmethod
     def display_name(self):
+        """
+        The name as it should be displayed
+        """
         pass
 
     @property
     @abstractmethod
     def isNull(self):
+        """
+        Is the null class?
```

Comment:
It is to test if the instance is an instance of the null class, `NullOffenceClassPresenter`, so that I can avoid `if None` checks in the templates. This may have been a bad decision.

`NullOffenceClassPresenter` is for the selected offence class it has not been selected while `NoneOffenceClassPresenter` is for a price where the offence class field is `None`. This really doesn't feel right as there are effectively two null classes for slightly different situations